modules/tinymce/src/core/demo/ts/demo/FullDemo.ts

```diff
@@ -119,10 +163,14 @@ export default (): void => {
       'searchreplace', 'wordcount', 'visualblocks', 'visualchars', 'code', 'fullscreen', 'insertdatetime', 'media', 'nonbreaking',
       'save', 'table', 'directionality', 'emoticons', 'importcss', 'codesample', 'help', 'accordion'
     ],
+    menubar: 'file edit view tools mode format table help',
+    menu: {
+      mode: { title: 'Mode', items: 'lock unlock' }
+    },
     // rtl_ui: true,
     add_unload_trigger: false,
     autosave_ask_before_unload: false,
-    toolbar: 'undo redo sidebar1 fontsizeinput | bold italic underline strikethrough | alignleft aligncenter alignright alignjustify | align lineheight fontsize fontfamily blocks styles insertfile | styles | ' +
+    toolbar: 'readonlyUIMode readonlySelectionMode design | undo redo sidebar1 fontsizeinput | bold italic underline strikethrough | alignleft aligncenter alignright alignjustify | align lineheight fontsize fontfamily blocks styles insertfile | styles | ' +
```

Comment:
Maybe we should clean these up, perhaps moved to the end of the toolbar

Reply:
Moved these changes to a separate demo. Changes in FullDemo are reverted.